app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -441,7 +441,7 @@ class HomeActivityTest {
   }
 
   @Test
-  fun testHomeActivity_recyclerViewIndex4_topicSummary_configurationChange_lessonCountIsCorrect() {
+  fun testHomeActivity_topicSummary_configurationChange_lessonCountIsCorrect() {
```

Comment:
let's use configChange to bring consistency

Reply:
done